images.yaml

```diff
@@ -374,6 +374,24 @@
     - tagSuffix: giantswarm
       dockerfileOptions:
       - RUN apt-get update && apt-get install -y ca-certificates && rm -rf /var/lib/apt/lists/*
+- name: k8s.gcr.io/kube-apiserver
+  tagTrimVersionPrefix: true
+  overrideRepoName: hyperkube
+  patterns:
+  - pattern: '>= v1.19.8 || >= v1.20.0-0'
```

Comment:
Starting with 1.20 I think it would be cool to get pre-release versions so we can get ahead of the release curve.